safe_transaction_service/safe_messages/tests/test_views.py

```diff
@@ -175,13 +175,33 @@ def test_safe_messages_create_view(self, get_owners_mock: MagicMock):
                     "message": message,
                     "description": description,
                     "signature": signature,
+                    "safeAppId": -1,
                 }
                 response = self.client.post(
                     reverse("v1:safe_messages:safe-messages", args=(safe_address,)),
                     format="json",
                     data=data,
                 )
                 self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
+                self.assertEqual(
+                    response.data,
+                    {
+                        "safe_app_id": [
+                            ErrorDetail(
+                                string="A positive integer value must be greater than 0",
```

Comment:
This message should also be updated right?

Reply:
Yes, done too!